src/strategy/dungeons/wotlk/forgeofsouls/FosStrategy.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _PLAYERBOT_WOTLKDUNGEONFOSSTRATEGY_H
+#define _PLAYERBOT_WOTLKDUNGEONFOSSTRATEGY_H
+#include "Multiplier.h"
+#include "Strategy.h"
+
+class WotlkDungeonFosStrategy : public Strategy
+{
+public:
+    WotlkDungeonFosStrategy(PlayerbotAI* ai) : Strategy(ai) {}
+    std::string const getName() override { return "fos"; }
```

Comment:
Better to expand this out to "forge of souls" - it's almost impossible to understand what "fos" would mean in the strategy list when you print out all the existing strategies, unless you happen to remember the dungeon acronyms